proto/pdpb.proto

```diff
@@ -1,303 +1,321 @@
-syntax = "proto2";
+syntax = "proto3";
 package pdpb;
 
 import "metapb.proto";
-import "eraftpb.proto";
 
 import "gogoproto/gogo.proto";
 
-option (gogoproto.marshaler_all) = true;
 option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
-message Leader {
-    optional string addr  = 1 [(gogoproto.nullable) = false];
-    optional uint64 id    = 3 [(gogoproto.nullable) = false];
-
-    // Due to nonuniqueness, pid is no longer used.
-    // optional int64  pid   = 2 [(gogoproto.nullable) = false];
-}
-
-enum CommandType {
-    Invalid             = 0;
-    Tso                 = 1;
-    Bootstrap           = 2;
-    IsBootstrapped      = 3;
-    AllocId             = 4;
-    GetStore            = 5;
-    PutStore            = 6;
-    AskSplit            = 7;
-    GetRegion           = 8;
-    RegionHeartbeat     = 9;
-    GetClusterConfig    = 10;
-    PutClusterConfig    = 11;
-    StoreHeartbeat      = 12;
-    ReportSplit         = 13;
-    GetRegionByID       = 14;
-    GetPDMembers        = 15;
+service PD {
```

Comment:
In rust, only first latter should use upper case. Like `RwLock` instead of `RWLock`, `Uuid` instead of `UUID`.